taker-frontend/src/components/NavBar.tsx

```diff
@@ -77,29 +90,48 @@ export default function Nav({ walletInfo, connectedToMaker, fundingRate, nextFun
                         </Menu>
                     </Flex>
                     <HStack>
-                        <Text>{"Maker status: "}</Text>
-                        <Heading size={"sm"}>{connectionMessage}</Heading>
-                        <Divider orientation={"vertical"} borderColor={"black"} height={"20px"} />
+                        <Text>{"Maker: "}</Text>
+                        <Heading
+                            size={"sm"}
+                            color={useColorModeValue(connectionMessage.color.light, connectionMessage.color.dark)}
+                        >
+                            {connectionMessage.label}
+                        </Heading>
+                        <TextDivider />
                         <Text>{"Next funding event: "}</Text>
                         <Skeleton
                             isLoaded={nextFundingEvent != null}
                             height={"20px"}
-                            minWidth={"50px"}
                             display={"flex"}
                             alignItems={"center"}
                         >
-                            <Heading size={"sm"}>{nextFundingEvent}</Heading>
+                            <Tooltip
+                                label={"The next time your CFDs will be extended and the funding fee will be collected based on the hourly rate."}
+                                hasArrow
+                            >
+                                <HStack minWidth={"260px"}>
+                                    <Heading size={"sm"}>{nextFundingEvent}</Heading>
+                                    <Text>{"@ a rate of"}</Text>
```

Comment:
% implies a rate so we could drop this "a rate of" text if you wanted to save some more space?